CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- `--tmp` parameter (optional)
```

Comment:
`tmp` is not clear enough about what this does. We usually call this kind of thing `build-dir` in other scrapers.

Also pleas include a brief summary of what it does here in the changelog